src/plugins/pluginProxyRemoteEntry.ts

```diff
@@ -24,12 +24,12 @@ export default function (): Plugin {
       if (id.includes(REMOTE_ENTRY_ID)) {
         return generateRemoteEntry(getNormalizeModuleFederationOptions());
       }
-      if (id.includes(WRAP_REMOTE_ENTRY_QUERY_STR)) {
-        return generateWrapRemoteEntry();
-      }
-      if (id.includes(HOST_AUTO_INIT_QUERY_STR)) {
-        return generateWrapHostInit();
-      }
+      // if (id.includes(WRAP_REMOTE_ENTRY_QUERY_STR)) {
```

Comment:
do we need this commented code

Reply:
Yes, it can be deleted. The current vite mechanism cannot use virtual modules, so these two lines of code can be omitted. I just think that vite6 and vite7 may be able to use virtual modules, so it can be restored.